src/uu/basename/src/basename.rs

```diff
@@ -11,13 +11,11 @@ use clap::{crate_version, Arg, ArgAction, Command};
 use std::path::{is_separator, PathBuf};
 use uucore::display::Quotable;
 use uucore::error::{UResult, UUsageError};
-use uucore::format_usage;
+use uucore::{format_usage, help_usage, help_about};
```

Comment:
rustfmt complains that `help_about` and `help_usage` should be ordered alphabetically